kodi_addon_checker/common.py

```diff
@@ -1,14 +1,11 @@
-def colorPrint(string, color):
-    print("\033[%sm%s\033[0m" % (color, string))
+import importlib
+import os
+import pkgutil
+import sys
 
 
-def check_config(config, value):
-    if config is None:
-        return False
-    elif value in config and config[value]:
-        return True
-    else:
-        return False
+def colorPrint(string, color):
```

Comment:
I think we should move this method to `console_reporter.py` as it's console specific.

Reply:
I thought about it but later didn't do that because it will create a dependency between addon-checker and `ConsoleReporter`. In my opinion, ConsoleReporter and whatever we have in `plugins` package should not be accessed directly from add-on checker.
If we move `colorPrint` to console reporter, I would recommend using normal `print` everywhere else.
WDYT?

Reply:
Sorry.
After seeing your other comments, now it makes sense. I am removing all `colorPrint` calls.